modules/tests/src/test/resources/testxml/general/TestCentroid.xml

```diff
@@ -242,6 +242,30 @@
 <test><op name="getCentroid" arg1="A" >    POINT (56.52883333335 25.21033333335)  </op></test>
 </case>
 
-
+<case>
+  <desc>A - almost degenerate MultiPolygon</desc>
+  <a>
+    MULTIPOLYGON (((
+     -92.661322 36.58994900000003,
+     -92.66132199999993 36.58994900000005,
+     -92.66132199999993 36.589949000000004,
+     -92.661322 36.589949,
+     -92.661322 36.58994900000003)),
+    ((
+     -92.65560500000008 36.58708800000005,
+     -92.65560499999992 36.58708800000005,
+     -92.65560499998745 36.587087999992576,
+     -92.655605 36.587088,
+     -92.65560500000008 36.58708800000005
+    )),
+    ((
+     -92.65512450000065 36.586800000000466,
+      -92.65512449999994 36.58680000000004,
+     -92.65512449998666 36.5867999999905,
+      -92.65512450000065 36.586800000000466
+    )))
+  </a>
+  <test><op name="getCentroid" arg1="A" >POINT (-92.6553838608954 36.58695407733924)</op></test>
```

Comment:
Since the XML Test format doesn't provide a way of expressing a tolerance value, this is fine as it stands.  If it breaks in the future, that will be motivation to add a tolerance!